R/Solver.R

```diff
@@ -9,161 +9,197 @@
 ### or it can be a matrix of column vectors to solve for.
 ### If Rank_Analysis has been done, the rank_an output can be inputted to avoid repeating rank-finding.
 ### DFT_obj is an output from the gen_DFT() function
-solve_WA <- function(DFT_obj,A_mat,v,
-                     rank_obj=NULL,DID_full=FALSE) {
+solve_WA <- function(
+    DFT_obj,
+    A_mat,
+    v,
+    rank_obj = NULL,
+    DID_full = TRUE
+) {
   ## If v is a vector, put it into matrix form:
   if (is.vector(v)) {
-    v <- matrix(data=v, ncol=1)
+    v <- matrix(data = v, ncol = 1)
```

Comment:
Infix operators (`+`) should be separated by spaces. Source: https://style.tidyverse.org/syntax.html#infix-operators